modules/specialized-guides/pages/salt/salt-ssh.adoc

```diff
@@ -27,8 +27,15 @@ Use this method for in-place firewall environments that block HTTP or HTTPS conn
 As with all Salt calls, {productname} invokes `salt-ssh` via the `salt-api`.
 
 Salt SSH relies on a roster to obtain details such as hostname, ports, and the SSH parameters of a client.
-{productname} keeps these details in the database and makes them available to Salt by generating a temporary roster file for each Salt SSH call.
-The location of the temporary roster file is supplied to `salt-ssh` using the `--roster-file= option`.
+{productname} keeps these details in the database and makes them available to Salt by using `uyuni` roster module or by generating a temporary roster file on bootstrapping new clients with {webui} using Salt SSH call.
+The location of the temporary roster file is supplied to `salt-ssh` using the `--roster-file=` option.
+For registered clients `--roster=uyuni` is used instead.
+
+[IMPORTANT]
+====
+It is not recommended to use [command]``salt-ssh`` as the call will be processed with current user permissions and in case of calling it with `root` user can cause permission issues for the files on processing next calls with `salt-api`.
+Please use [command]``mgr-salt-ssh`` instead. It is changing effective user to `salt` on each call. If you want to use it with a user other than `root`, the user should have the permission to change effective user to `salt`.
```

Comment:
```suggestion
Use [command]``mgr-salt-ssh`` instead because it is changing effective user to `salt` on each call.
If you want to use it with a user other than `root`, the user should have the permission to change effective user to `salt`.
```